src/main/java/rex/util/Ui.java

```diff
@@ -13,14 +13,20 @@
  * displaying messages, prompts, and error messages.

```

Comment:
Perhaps could start with a capital letter for consistency, as some comments start with capital letters while others start with lower case

Reply:
Nice spot, thank you!